app/javascript/components/workflow-repository-form/workflow-repository-form.schema.js

```diff
@@ -0,0 +1,88 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import { customUrlValidator } from './helpers';
+
+const getCredentials = () => API.get(
+  // eslint-disable-next-line max-len
+  '/api/authentications?collection_class=ManageIQ::Providers::Workflows::AutomationManager::ScmCredential&expand=resources&sort_by=name&sort_order=ascending'
```

Comment:
@agrare I know we only have one credential type at the moment, but should there be a base class for this that might be a  better choice?

Reply:
(total aside - `collection_class`?  I thought it was supposed to be `type`?)

Reply:
There is a base class, `ManageIQ::Providers::Workflows::AutomationManager::Credential` but since this is the repository form I think he actually does only want `ScmCredentials`